examples/demo-app/src/actions.js

```diff
@@ -380,12 +385,25 @@ export function exportFileToCloud(providerName) {
     .then(
       response => {
         if (cloudProvider.shareFile) {
-          dispatch(push(`/${MAP_URI}${response.url}`));
+          const responseUrl = (cloudProvider.getMapPermalink)
```

Comment:
why parenthesis?

Reply:
This is just to emphasize that it is a ternary assignment, but it can be removed.